src/Microsoft.ML.TimeSeries/ExtensionsCatalog.cs

```diff
@@ -187,14 +187,15 @@ public static class TimeSeriesCatalog
         /// It is used when score is calculated for each root cause item. The range of beta should be in [0,1].
         /// For a larger beta, root cause items which have a large difference between value and expected value will get a high score.
         /// For a small beta, root cause items which have a high relative change will get a low score.</param>
+        /// <param name="rootCauseThreshold">A threshold to determine whether the point should be root cause. If the point's delta is equal to or larger than rootCauseThreshold multiplied by anomaly dimension point's delta, this point is treated as a root cause. Different threshold will turn out different results. Users can choose the delta according to their data and requirments. </param>
         /// <example>
```

Comment:
What is the range of valid values for this parameter?

Reply:
> What is the range of valid values for this parameter?

Have updated the range